package.json

```diff
@@ -550,7 +550,7 @@
     },
     "configuration": {
       "type": "object",
-      "title": "Azure IoT Explorer configuration",
+      "title": "Azure IoT Hub Toolkit configuration",
```

Comment:
Is it as designed to change the configuration set name?

Reply:
It's just a title, all configuration key (e.g. azure-iot-toolkit.iotHubConsumerGroup) remain unchanged

Reply:
I update this title because when I tried to find configurations the first time, I seek "IoT Toolkit" keyword through my eyes and missed the "IoT Explorer" part. There's inconsistency between configuration title and extension name.